<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_event</class>
 <widget class="QDialog" name="Dialog_event">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pB_AjouterP">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_ModifierP">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_SupprimerP">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_EventID">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    background-color: rgba(134, 47, 147, 50);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gB_infopersonnel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(199, 153, 207, 200);
background: rgba(199, 153, 207, 100); 
border-radius: 10px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_Heure">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Heure</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Nom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_EventID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Event ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Lieu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Lieu</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EventID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Type">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Lieu">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Date">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_DateE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    background-color: rgba(134, 47, 147, 50);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}</string>
    </property>
    <property name="displayFormat">
     <string>dd-MM-yyyy</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_HeureE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTimeEdit
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    background-color: rgba(134, 47, 147, 50);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NbPersonnes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Nbr des personnes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NbPersonnes">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tab_Events">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>801</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TirerParNbPersonnes">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>590</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par Nb.Personnes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SupprimerE">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Fermer">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>650</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(189, 96, 96, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_Events">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>120</y>
     <width>341</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TirerParNom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par Nom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TireParType">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>590</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_TitleEvents">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LEMON MILK</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147,50);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
   </property>
   <property name="text">
    <string>GESTION DES EVENTMENTS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_BGPersonnel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/Image/Images/BGSponsoring.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147, 25);
	border: 2px solid rgba(134, 47, 147, 100);
		border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_BGPersonnel</zorder>
  <zorder>pB_AjouterP</zorder>
  <zorder>pB_ModifierP</zorder>
  <zorder>pB_SupprimerP</zorder>
  <zorder>comboBox_EventID</zorder>
  <zorder>gB_infopersonnel</zorder>
  <zorder>tab_Events</zorder>
  <zorder>pB_TirerParNbPersonnes</zorder>
  <zorder>pushButton_SupprimerE</zorder>
  <zorder>pushButton_Fermer</zorder>
  <zorder>text_Events</zorder>
  <zorder>pB_TirerParNom</zorder>
  <zorder>pB_TireParType</zorder>
  <zorder>labe_TitleEvents</zorder>
  <zorder>label_Info</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
